src/plugins/intel_cpu/src/nodes/mkldnn_normalize_node.cpp

```diff
@@ -36,6 +37,47 @@ using namespace Xbyak;
 
 #define THROW_ERROR IE_THROW() << "NormalizeL2 layer with name '" << getName() << "' "
 
+namespace {
+struct NormalizeKey {
+    MKLDNNNormalizeL2Node::NormalizeL2Attrs attrs;
+    mkldnn::primitive_attr kernel_attrs;
+    VectorDims dims;
+
+    size_t hash() const;
+    bool operator==(const NormalizeKey& rhs) const;
+};
+
+size_t NormalizeKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+    seed = hash_combine(seed, attrs.epsMode);
+    seed = hash_combine(seed, attrs.across_spatial);
+    seed = hash_combine(seed, attrs.cornerCase);
+    seed = hash_combine(seed, attrs.eps);
+    seed = hash_combine(seed, attrs.is_nchw);
+    seed = hash_combine(seed, attrs.is_nhwc);
+    seed = hash_combine(seed, attrs.is_blk);
```

Comment:
May I ask you to refactor the code a little bit: we can use single enum value instead of 3 separate bool flags. It should speedup hash generation a little bit.
See as an example: https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/nodes/mkldnn_interpolate_node.h#L18)

Reply:
Good idea, changed. Thanks!